libs/payments/stripe/src/lib/stripe.manager.ts

```diff
@@ -146,4 +148,22 @@ export class StripeManager {
 
     return taxIdFields.at(0)?.value;
   }
+
+  async getPlan(planId: string) {
+    const plan = await this.client.plansRetrieve(planId);
+    if (!plan) throw new PlanNotFoundError();
+    return plan;
+  }
+
+  async getPlanByInterval(planIds: string[], interval: string) {
```

Comment:
This is currently a placeholder method, and a new ticket will be added to flesh this out more, so that a CMS Interval, `daily`, `weekly`, `6monthly`, `yearly`, etc. is correctly mapped to the Stripe `interval` and `interval_count`.

Reply:
To be completed by [FXA-9536](https://mozilla-hub.atlassian.net/browse/FXA-9536)